base/docs/helpdb/Base.jl

```diff
@@ -1437,10 +1437,16 @@ endof
 Constructs a `Channel` that can hold a maximum of `sz` objects of type `T`. `put!` calls on
 a full channel block till an object is removed with `take!`.
 
+`Channel(0)` constructs a Channel without a backing store. Consequently a `put!` on a
+0-sized channel will block till another task calls a `take!` on it. And vice-versa.
+
+`isready` on a 0-sized channel returns true if there are any tasks blocked on a `put!`
```

Comment:
should be separated with either a period or another line between this and the following statement